### PR TITLE
Update PR for LWG3769

### PR DESCRIPTION
**Pull request by tomaszkam**

### CODE REVIEW

**Comment by tomaszkam** [comment]
In case of any issues with the PR please let me know or fix it locally. Whatever is more convenient for you.

**Comment by Dani-Hub** [comment]
In take a look this weekend and will presumably just accept the PULL request and fix any issues on my side

**Comment by Dani-Hub** [comment]
@tomaszkam I have fixed quite a number of copy&paste errors, please reload and double-check the new wording state:
https://cplusplus.github.io/LWG/issue3769

File: xml/issue3769.xml [hunk 1]
```diff
@@ -56,6 +56,108 @@ Set priority to 1 after reflector poll.
...
+</ol>
+</blockquote>
+
+<note>2022-11-04; Tomasz comments and improves proposed wording</note>
+
+<p>
+Initially, LWG requested an investigation of alternative resolutions that would avoid using member functions for the affected operators.
+Later, it was found that in addition to <tt>==</tt>/<tt>-</tt>, all comparison operators (<tt>&lt;</tt>, <tt>&gt;</tt>, <tt>&lt;=</tt>, <tt>&gt;=</tt>, <tt>&lt;=&gt;</tt>) are affected by same problem for the calls
+with <tt>basic_const_iterator&lt;basic_const_iterator&lt;int*&gt;&gt;</tt> and <tt>int*</tt> as arguments, i.e. <tt>totally_ordered_with&lt;basic_const_iterator&lt;basic_const_iterator&lt;int*&gt;&gt;, int*&gt;</tt>
+causes infinite recursion in contrast checking.
```
Comment (JohelEGP) [inline xml/issue3769.xml:143]: Did you mean "constraint checking" here?

Comment (Dani-Hub) [reply xml/issue3769.xml:143]: I fixed that now (including two other typos)

**Comment by tomaszkam** [comment]
Looks good. Thanks!

**Pull request merged**